### ISSUE TITLE
Hyper-parameters for batch-size 16k

### ISSUE THREAD

**Issue by yangorwell** [binmakeswell/LARS-ImageNet-PyTorch#2]
Hi,
I meet a problem and look forward to you suggestion!

I am trying to see the performance for batch-size 16k. I use 4 gpus and  run the command as below:

horovodrun -np 4 python pytorch_imagenet_resnet.py  --epochs 90 --model resnet50 --batch-size 256 --warmup-epochs 5 --train-dir=/mnt/ILSVRC2012/train/ --val-dir=/mnt/ILSVRC2012/val --base-lr 22.627416997969522 --base-op lars --bn-bias-separately --wd 0.0001 --lr-scaling keep --batches-per-allreduce 16

The performance in the first 3 epochs seems very bad where the top-1 validation accuracy is less than 10%. Do you ever test such large batch size? And how do you do in that setting? 

Best Regards,
Minghan.

**Comment by binmakeswell** [comment]
Hi, Minghan,

I have tested the performance for batch-size 16k and it also performs well. The top-1 validation accuracy about 10%+ at 3rd epoch and can reach around 77% finally. 

I use similar hyper-parameters while the batch size is 128 on each GPU. I not sure the performance when the batch size is 256 on each GPU, it will affect the performance of Batch Normalization at least.

Hope my answer can help you : )

**Comment by yangorwell** [comment]
Thanks for your reply !!
After running LARS for 90 epochs under my command I also get the similar results. 
Thank you again for the helpful reply!
